content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst

```diff
@@ -28,24 +28,24 @@ three-step receipts. First, make sure the *Multi-Step Routes* option is enabled
 Next, the warehouse needs to be configured for three-step receipts. Go to :menuselection:`Inventory
 --> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the warehouse. Then, select
 :guilabel:`Receive goods in input, then quality and then stock (3 steps)` for :guilabel:`Incoming
-Shipments`. Then, :guilabel:`Save` the changes.
+Shipments`.
 
 .. image:: receipts_three_steps/three-step-receipt-settings.png
    :align: center
    :alt: Set incoming shipment option to receive in three steps.
 
 Activating three-step receipts and deliveries will create two new internal locations:
 *Input* (WH/Input), and *Quality Control* (WH/Quality Control). To rename these locations, go to
-:menuselection:`Configuration --> Locations`, select the :guilabel:`Location` to change, click
-:guilabel:`Edit`, update the name, and finally, click :guilabel:`Save`.
+:menuselection:`Configuration --> Locations`, click on the :guilabel:`Location` to change, and
+update the name.
 
 Process a receipt in three steps (input + quality + stock)
 ==========================================================
 
```

Comment:
Could be nice to add a quick 1-2 sentence intro into the section that follows - just a suggestion

Reply:
This is also explained in the very beginning why this would be beneficial. I could move it, but I wanted some kind of introduction at the top, and didn't want to say it twice.